Dockerfile

```diff
@@ -9,10 +9,7 @@ COPY *.json ./
 RUN npm install --ignore-scripts
 
 # Copy all source code
-COPY src ./src
-
-# Build the application
-RUN npm run build
+COPY . .
```

Comment:
This is so that as the project grows you don't have to constantly update a file list. You are free to change this if you prefer.